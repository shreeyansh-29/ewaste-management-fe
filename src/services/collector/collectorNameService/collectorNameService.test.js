/* eslint-disable no-undef */
import React from "react";
import Enzyme, {shallow} from "enzyme";
import {collectorNameService} from "./collectorNameService";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

it("testing collectorNameService", () => {
  const Service = collectorNameService;
  const wrapper = shallow(<Service />);
  expect.assertions(1);
  expect(toJson(wrapper)).toMatchSnapshot();
});
