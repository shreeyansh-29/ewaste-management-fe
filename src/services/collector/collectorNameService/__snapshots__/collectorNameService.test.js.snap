// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing collectorNameService 1`] = `<Component />`;
